testplan/web_ui/testing/src/Toolbar/Toolbar.js

```diff
@@ -282,6 +310,7 @@ class Toolbar extends Component {
             {this.filterButton(toolbarStyle)}
             {this.printButton()}
             {this.tagsButton()}
+            {this.timeButton()}
```

Comment:
As mentioned above, let's move this timeButton out of the toolbar and make it an extraButton passed in by the BatchReport only - it doesn't make sense to have a button that does nothing in interactive UI.

Reply:
I remove it from interactive UI, place it in extraButtons.